src/patternfly/demos/Table/examples/Table.md

```diff
@@ -4,6 +4,8 @@ section: components
 wrapperTag: div
 ---
 
+import './Table.css'
```

Comment:
Does this file exist? I don't see it - https://github.com/patternfly/patternfly/tree/37f003a829d05765706337e0b95075ac5bf4d15c/src/patternfly/demos/Table/examples